examples/http/src/server.rs

```diff
@@ -22,20 +19,22 @@ async fn handle(req: Request<Body>) -> Result<Response<Body>, Infallible> {
     Ok(Response::new("Hello, World!".into()))
 }
 
-fn init_tracer() -> impl Tracer {
+fn init_tracer() {
     global::set_text_map_propagator(TraceContextPropagator::new());
 
     // Install stdout exporter pipeline to be able to retrieve the collected spans.
     // For the demonstration, use `Sampler::AlwaysOn` sampler to sample all traces. In a production
     // application, use `Sampler::ParentBased` or `Sampler::TraceIdRatioBased` with a desired ratio.
-    stdout::new_pipeline()
-        .with_trace_config(trace::config().with_sampler(Sampler::AlwaysOn))
-        .install_simple()
+    let provider = TracerProvider::builder()
```

Comment:
nits: The comment above about AlwaysOnSampler being used is no longer relevant.

Reply:
Yeah can probably remove now